### ISSUE TITLE
Beta UI: column headers SERP not accessible to screen readers

### ISSUE THREAD

**Issue by klml** [zammad/zammad#6164]
### Used Zammad Version

7.0.1-df77241a.docker

### Environment

Openshift

### Actual behaviour

The column headers in the search results list are not accessible to screen readers and are not read aloud:

<img width="1632" height="207" alt="Image" src="https://github.com/user-attachments/assets/3c157954-2636-428a-8d28-d4fe65554916" />

### Expected behaviour

The column headers in the search results list should be read aloud by the screen reader

### Steps to reproduce the behaviour

Use Screenreader

### Support Ticket

Ticket#10226081

### I'm sure this is a bug and no feature request or a general question.

no
